.github/workflows/lint-autoformat.yml

```diff
@@ -0,0 +1,46 @@
+name: Apply lint suggestions
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+    branches: [main]
+
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [gh/**/base]
+
+jobs:
+  lintrunner-autoformat:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: lf.linux.2xlarge
+    if: ${{ github.repository_owner == 'pytorch' }}
+    steps:
+      - name: Checkout PyTorch
+        uses: pytorch/pytorch/.github/actions/checkout-pytorch@main
+        with:
+          submodules: true
+          fetch-depth: 0
+      - name: Setup miniconda
```

Comment:
Should we use actions/setup-python instead of conda? I think we want to move away from conda in CI.

Reply:
I think I might know why.  IIRC, `setup-python` unfortunately didn't work with our runner as it looked for the distro release file (/etc/release?), and it didn't understand the file from AmazonLinux2.  We could try to use setup-python again now with the new AmazonLinux2023 and the latest version of actions/setup-python